src/provider/view/containers/ToastApp.tsx

```diff
@@ -51,9 +51,14 @@ const mapStateToProps = (state: RootState, ownProps: ToastAppProps) => ({
 
 const Container = connect(mapStateToProps)(ToastApp);
 
-export function renderApp(notification: StoredNotification, document: Document, store: Store, setWindowSize: (dim: WindowDimensions) => void) {
+export function renderApp(
+    notification: StoredNotification,
+    document: Document,
+    store: ServiceStore,
+    setWindowSize: (dim: WindowDimensions) => void
+) {
     ReactDOM.render(
-        <Provider store={store['_store']}>
+        <Provider store={store as any}>
```

Comment:
So does React still view our store as a Redux store? This might be something I need explaining to me in person 😛

(Same for NotificationCenterApp.tsx)

Reply:
Our store implements the 3 methods `getState`, `subscribe` & `dispatch` so it does recognize it. The reason `any` is that the `<Provider>` expects a very strict definition of `Dispatch<A>` that is a nightmare to try and match. It's the same reason why before it used `store['store']` instead of just store.

Reply:
Suggest do this as something like `store as unknown as Store<RootState, RootAction>` to communicate what type we're pretending `store` is, and put a comment why this is needed

Reply:
that won't resolve the interface compatibility issue.

Reply:
I'm suggesting this purely for communication, so there's some indication in code what type we're pretending `store` is

Reply:
oh no, I mean it won't resolve the interface issue we have in react-redux :)  store as unknown as Store<RootState, RootAction> still causes the transpiler to complain about the dispatch method because we're trying to fit our class into react definition.

Reply:
Doing `import {Store} from 'redux'` then `<Provider store={store as unknown as Store<RootState>}>` works fine for me

Reply:
sorry, Store<RootState, RootAction> bit confused me as that's the signature for our own Store implementation class. we have too many stores now :)

Reply:
@bryangaleOF But we're going to remove the `redux` package?

Reply:
`react-redux` depends on it, so it'll be in our dependency tree anyway

Reply:
It has a `peerDependency` & `devDependency` on `redux` so it's not in our dependency tree. Its the same situation as with `openfin-service-signal`, the `peerDependency` is just so the types match up if we were to use `redux`.

Reply:
It does seem that it gets pulled in somehow. Which, stands to reason I think? If there's a type definition for the transpiler to complain that we're not fulling, there's a type definition for us to cast to.

My basic point is, we're pretending and object is type X, so let's cast to type X to make it clear what we're doing, rather than casting to `any`.